task_and_robot_environment_name: 'SingleAgentEvaluation-v1'
num_envs: 1
neural_nw_path: "/drl_ws/logs/1.1/best_model.pkl"



